booking-app/src/main/java/org/motechproject/bookingapp/domain/VisitBookingDetailsDto.java

```diff
@@ -0,0 +1,150 @@
+package org.motechproject.bookingapp.domain;
+
+import org.codehaus.jackson.map.annotate.JsonDeserialize;
+import org.codehaus.jackson.map.annotate.JsonSerialize;
+import org.joda.time.LocalDate;
+import org.motechproject.bookingapp.util.CustomDateDeserializer;
+import org.motechproject.bookingapp.util.CustomDateSerializer;
+import org.motechproject.bookingapp.util.CustomTimeSerializer;
+import org.motechproject.commons.date.model.Time;
+import org.motechproject.ebodac.domain.Gender;
+import org.motechproject.ebodac.domain.Subject;
+
+public class VisitBookingDetailsDto {
```

Comment:
I don't think this should be named VisitBookingDetailsDto is more something like PrimeVaccinationScheduleDto to me